ecommerce_api_client/auth.py

```diff
@@ -1,24 +1,40 @@
+# pylint: disable=duplicate-code
```

Comment:
Is this still necessary? When run locally, I see no errors with both of these duplicate-code statements removed.